ksqldb-engine/src/main/java/io/confluent/ksql/engine/QueryPlan.java

```diff
@@ -33,12 +33,14 @@
   private final Optional<SourceName> sink;
   private final ExecutionStep<?> physicalPlan;
   private final QueryId queryId;
+  private final Optional<String> usesSharedRuntime;
```

Comment:
super nit: name makes it sound like this is a boolean, can you describe what it's actually supposed to be used for/mean?